modules/sdk/src/main/scala/org/tessellation/sdk/http/p2p/clients/L0TrustClient.scala

```diff
@@ -0,0 +1,30 @@
+package org.tessellation.sdk.http.p2p.clients
+
+import cats.effect.Async
+
+import org.tessellation.ext.codecs.BinaryCodec._
+import org.tessellation.kryo.KryoSerializer
+import org.tessellation.schema.peer.L0Peer
+import org.tessellation.schema.trust.TrustScores
+import org.tessellation.sdk.domain.trust.storage.OrdinalTrustMap
+import org.tessellation.sdk.http.p2p.PeerResponse
+import org.tessellation.security.SecurityProvider
+
+import org.http4s.client.Client
+
+trait L0TrustClient[F[_]] {
+  def getLatestTrust: F[TrustScores]
+  def getLatestDeterministicTrust: F[OrdinalTrustMap]
+}
+
+object L0TrustClient {
+  def make[F[_]: Async: SecurityProvider: KryoSerializer](client: Client[F], globalL0Peer: L0Peer): L0TrustClient[F] =
+    new L0TrustClient[F] {
+
+      def getLatestTrust: F[TrustScores] =
+        PeerResponse[F, TrustScores]("trust/latest")(client).run(globalL0Peer)
+
+      def getLatestDeterministicTrust: F[OrdinalTrustMap] =
```

Comment:
The path changed but you didn't change the method name. Should it stay like that or should it be getCurrentOrdinalTrust?

Reply:
Changed to `getLatestCurrentOrdinalTrust`.